src/components/pageComponents/ExpandableListItem.js

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import {
+  ListItem,
+  ListItemText,
+  Collapse,
+  makeStyles,
+} from "@material-ui/core/";
+import { ExpandLess, ExpandMore } from "@material-ui/icons/";
```

Comment:
Linebreaks for multiple imports (even for two - I think we should keep it consistent)